client/src/components/SignupForm/SignupForm.js

```diff
@@ -9,22 +9,35 @@ import styles from '../LoginForm/LoginForm.scss'
 import { signup } from '../../actions'
 
 function EmailField ({ input, meta: { error } }) {
+  var classNames = require('classnames')
+  const emailClasses = classNames(styles.inputEmailContainer, {
+    [styles.error]: error,
+    [styles.valid]: !error
+  })
```

Comment:
Instead of introducing a new npm library that may increase the bundle size and time, could we do something like this?

```js
const emailClasses = `${styles.inputEmailContainer} ${error ? styles.error : styles.valid}`
```

It's shorter and only one line! :) We can do this for also the PasswordField.

Reply:
It's done :) I didn't know that syntax and searching in google everyone pointed to that dependency.
thanks